jobs/integr8ly/pds-testing-executor.yaml

```diff
@@ -168,6 +168,7 @@ This needs to be changed to 'false' for Integreatly Workshop since it uses valid
 
             if (testPipeline == 'after-first-login-tests' || testPipeline == 'browser-based-testsuite-pipeline') {
                 parameters.add(string(name: 'EVALS_USERNAME', value: 'evals23@example.com'))
+                parameters.add(string(name: 'EVALS_PASSWORD', value: 'Password1'))
```

Comment:
I think you should additionally remove the EVALS_PASSWORD from the next 'if statement'

Reply:
OK good point I think I will rather remove the or condition (I think it will be more clear)